x-pack/plugins/monitoring/server/kibana_monitoring/bulk_uploader.js

```diff
@@ -52,11 +52,11 @@ export class BulkUploader {
     });
 
     this.kibanaStats = kibanaStats;
-    this.kibanaStatusGetter = null;
-  }
 
-  setKibanaStatusGetter(getter) {
-    this.kibanaStatusGetter = getter;
+    this.kibanaStatus = null;
+    this.kibanaStatusGetter$ = statusGetter$.subscribe((nextStatus) => {
+      this.kibanaStatus = nextStatus.level.toString();
```

Comment:
nit: instead of converting to a string, you could import `ServiceStatusLevels` from `core/server` and do comparisons below:

```
if (this.kibanaStatus === ServiceStatusLevels.available) { }
// etc
```

This will make refactoring easier if we ever decide to change the names of these.

Reply:
Yes, great point. I had assumed incorrectly I couldn't import it because it was a typescript interface but it's just a constant.